provider/resources.go

```diff
@@ -124,6 +124,32 @@ func Provider() tfbridge.ProviderInfo {
 			"artifactory_release_bundle_v2_promotion": {ComputeID: computeIDField("name")},
 			"artifactory_item_properties":             {ComputeID: computeIDField("repoKey")},
 
+			// ComputeID mappings for v12.2.0 on webhooks
+			"artifactory_artifact_custom_webhook":                    {ComputeID: computeIDField("id")},
+			"artifactory_user_custom_webhook":                        {ComputeID: computeIDField("id")},
```

Comment:
Likewise for `"artifactory_user_custom_webhook"`.
```suggestion
			"artifactory_user_custom_webhook":                        {ComputeID: computeIDField("key")},
```